tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
Ok, these should be hard-coded and go in the test files themselves.  Generally the .txt files distributed with the tests are for regression testing.  Did you end up getting any ground-truth transcriptions + algorithm output?  If so, please replace with that.